### PR TITLE
Allow `use` for `#[pyattr]`

### PR DESCRIPTION
**Pull request by youknowone**

### CODE REVIEW

File: derive/src/pymodule.rs [hunk 1]
```diff
@@ -173,33 +173,22 @@ impl Module {
...
-                        }
-                        ItemType::Const => {
-                            items.push((
-                                Self::extract_const_attr(item.ident, into_nested()?)?,
-                                meta_span,
-                            ));
-                        }
-                        ItemType::Struct => {
+                    let item = match item.typ {
+                        ItemType::Fn => Self::extract_evaluated_attr(item.ident, into_nested()?)?,
```
Comment (coolreader18) [inline derive/src/pymodule.rs:177]: You could probably just do `Const | Use => ...` here

File: derive/src/pymodule.rs [hunk 2]
```diff
@@ -399,28 +388,46 @@ pub fn impl_pymodule(attr: AttributeArgs, item: Item) -> Result<TokenStream2, Di
...
+                    typ: ItemType::Const,
+                    attrs,
+                    ident,
+                },
+                Item::Use(syn::ItemUse { attrs, tree, .. }) => {
+                    let ident = match tree {
+                        UseTree::Path(path) => match &*path.tree {
+                            UseTree::Name(name) => &name.ident,
+                            UseTree::Rename(rename) => &rename.rename,
+                            _ => return None,
```
Comment (coolreader18) [inline derive/src/pymodule.rs:418]: It'd probably be nice to support multiple imports `use foo::{BAR, BAZ, QUX};`, though that would probably require a bit of restructuring other code to do so

Comment (youknowone) [reply derive/src/pymodule.rs:418]: I am thinking of overhaul of derive flow 😞

File: derive/src/pymodule.rs (hunk 1 above)
Comment (youknowone) [reply derive/src/pymodule.rs:177]: i will fix this one in the next PR

**Pull request merged**